.github/workflows/BuildImage.yml

```diff
@@ -15,8 +15,15 @@ jobs:
 
       - name: Build image
         run: |
-          docker build --no-cache -t ${{ github.sha }} .
-
+          # Build variables
+          VERSION=$(git describe --tags --always)
```

Comment:
This wont return anything useful

Reply:
It'll have the first 5 chars of the commit string.
I have this show up in the log on container start, mainly so I can keep track of future bug reports.  But I'm totally open to a better way to tag my builds.

Reply:
I would just use the github.sha, and truncate it, if you dont need the whole string. ie `${GITHUB_SHA:0:5}`

Reply:
Got it.  Updated as requested.
I'll work on a better, easier to maintain human-readable versioning scheme later.